### DISCUSSION TITLE
Version 2.2.0

### DISCUSSION THREAD

**Discussion by shazzaam7** [xenia-manager/xenia-manager#217] [General]
# Xenia Manager v2.2.0 Release 🎉  
**Happy New Year!** Thank you for your continued support.  

## 📝 Important Notes  
- **Compatibility Notice:** Versions 2.0.0 and later are not compatible with v1 of Xenia Manager. If you are updating from v1 to v2, please perform a full fresh reinstall.  

### **Game Parsing Without Xenia** (#214)  
- Introduced a new method for parsing game details that does not rely on Xenia.  
- This feature is a lightweight port of the [Xbox360Toolkit](https://github.com/Team-Resurgent/Xbox360Toolkit).  
- To use, enable the `Automatic Game Parsing and Selection` option in the settings.  
- Fallback: If `titleid` or `mediaid` cannot be retrieved, the system will revert to the previous parsing method.  
- **Supported Formats:**  
  - STFS  
  - XEX  
  - ISO  
  - GOD (limited to main file reading; `.data` files are unsupported but sufficient for Xenia Manager needs).  
- **Future Plans:** Pending bug fixes, this feature may evolve into a `Scan Directory` tool, similar to PR #191 by @JakariaMods.  

### **Optimized Game Settings**  
- Thanks to @A1eNaz, the list of games with optimized settings now includes **200 titles**!  
- View the complete list [here](https://github.com/xenia-manager/Optimized-Settings).  

## 🛠️ Bug Fixes  
- **Outdated URLs for GameInfo:** Resolved an issue where old URLs were being used to fetch GameInfo.  
- **Profiles Not Saving** (#215): Fixed an issue causing logged-in profiles not to save when closing the game due to `CreateSymbolicLink` failures.  

<hr /><em>This discussion was created from the release <a href='https://github.com/xenia-manager/xenia-manager/releases/tag/2.2.0'>Version 2.2.0</a>.</em>

**Comment by knight-ki11er** [comment]
Can you add a separate nvidia v-sync and frame limit options like in 1.15.3?

**Reply by shazzaam7** [reply]
Currently only Xenia's `vsync` and `framerate_limit` are implemented. I was going to implement NVIDIA stuff too, but got sidetracked. Will have to check it out whenever I can.

**Reply by knight-ki11er** [reply]
Thank you, it quite useful with games that capped at 30fps

**Reply by PedroTerraMoreira** [reply]
Hello, i see that Condemned 2 isn't on the compablity list, why that and it possible to make compablity?

**Reply by shazzaam7** [reply]
Xenia Manager uses the Xenia Canary's [compatibility list](https://github.com/xenia-canary/game-compatibility/)

**Reply by TheACR2024** [reply]
Where is then save directory of Xenia Manager?

**Reply by shazzaam7** [reply]
> Where is then save directory of Xenia Manager?

For automatic game files? It's in the Backup folder next to the executable.

**Comment by Githulp** [comment]
1) 2.0 release now triggers UAC alerts, previous 1.0 releases did not.  Is it possible to adjust it so it doesn't need UAC permissions?

2) 2.0 release cannot recognize mapped network drives.  Not sure if this was intentional, or related to UAC permissions but a fix for that would be very much appreciated.  Thank you

**Reply by Bobble80** [reply]
> 1. 2.0 release now triggers UAC alerts, previous 1.0 releases did not.  Is it possible to adjust it so it doesn't need UAC permissions?
> 2. 2.0 release cannot recognize mapped network drives.  Not sure if this was intentional, or related to UAC permissions but a fix for that would be very much appreciated.  Thank you

I came here to ask the same thing. Is it possible for the XM dev to look into fixing that?
When I tried V1 version of Xenia Manager, there was no UAC prompts when running the app, but now they’re in V2 of the app.

**Reply by shazzaam7** [reply]
UAC is required for Xenia Manager to be able to create Symbolic Links. This is an alternative take to "--config" that bypasses some of the issues with the "--config" argument (like the configuration file not updating on a run). This is how Windows is configured ans I can't do much about it other than to try to do something else instead of Symbolic Links.

**Reply by NoDowt** [reply]
Just came across this one too & was coming here to see why...

Not sure what issues you had with the previous method using the --config argument for xenia, as i don't really use it much (just to play Import Tuner Challenge occasionally); however needing to run As Admin is a bit of a risk from a security point of view. Should malicious code ever make it into Xenia Manager or Xenia (or any other child processes) it'll have free reign on the system.

I went back to using v1 for this reason.

Rather than using SymLinks, could you just copy files back & forth? Keep the config for each game in its own files & then before launching Xenia, you copy the file next to xenia exe... After Xenia exe is closed, copy the file back.

**Reply by shazzaam7** [reply]
Yeah, I'm going to be doing that from the next update onwards.

**Comment by NoDowt** [comment]
Just updated to v2 (temporarily, see other comment above) tonight without realising it should be done as a fresh install.

Something I was trying to work out, where were the config & patch files being saved now? (so i could pull the game settings across from v1)

Also, with the downloadable patches & suggested settings, how would I go about contributing to these?
Found a slightly different patch setting for Import Tuner Challenge which seemed to be more stable for me. And I've got a custom xenia config for it which seems to be working well for improving the look of it, collected a bunch of suggested settings found across reddit etc... Kept the ones which worked well for me.

**Reply by shazzaam7** [reply]
All of the optimized settings used by Xenia Manager are in this [repo](https://github.com/xenia-manager/Optimized-Settings). @A1eNaz is the one who maintains both it and the wiki for recommended settings.

As for the patches, Xenia Manager grabs them from Xenia Canary's official patches [repo](https://github.com/xenia-canary/game-patches)

Configuration and patches are stored in the same spot (`config` folder for configuration files (should be named as the game) and `patches` folder for patches (this is the same as v1 if I'm not mistaken))

**Reply by NoDowt** [reply]
Interesting, when I was running v2 I’m pretty sure it didn’t pickup the previous the config (was running from same location as v1); and when I did the auto-down load patch, it was showing that patch rather than my previously modified one; yet the file in patches folder (and showing when I rolled back to v1) was my previously modified patch.

**Reply by shazzaam7** [reply]
Because the emulators are in the different folder called `Emulators`.
